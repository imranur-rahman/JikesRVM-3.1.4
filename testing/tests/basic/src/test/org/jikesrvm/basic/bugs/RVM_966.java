/*
 *  This file is part of the Jikes RVM project (http://jikesrvm.org).
 *
 *  This file is licensed to You under the Eclipse Public License (EPL);
 *  You may not use this file except in compliance with the License. You
 *  may obtain a copy of the License at
 *
 *      http://www.opensource.org/licenses/eclipse-1.0.php
 *
 *  See the COPYRIGHT.txt file distributed with this work for information
 *  regarding copyright ownership.
 */
package test.org.jikesrvm.basic.bugs;

/**
 * Tests that the VM terminates with an error exit
 * code when an application thread is terminated due
 * to an uncaught expection.
 */
public class RVM_966 {
  public static void main(String[] args) {
    Object obj = null;
    System.out.println(obj.hashCode());
  }
}
