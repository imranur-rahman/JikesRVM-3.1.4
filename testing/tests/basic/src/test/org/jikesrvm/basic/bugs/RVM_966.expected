Exception in thread "MainThread" java.lang.NullPointerException
   at test.org.jikesrvm.basic.bugs.RVM_966.main(RVM_966.java:23)
